modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -660,13 +653,15 @@ impl ServerModule for Wallet {
             .add_items(dbtx, &UTXOPrefixKey, |_, v| v.amount.to_sat() as i64 * 1000)
             .await;
         audit
-            .add_items(dbtx, &UnsignedTransactionPrefixKey, |_, v| {
-                v.change.to_sat() as i64 * 1000
+            .add_items(dbtx, &UnsignedTransactionPrefixKey, |_, v| match v.rbf {
+                None => v.change.to_sat() as i64 * 1000,
+                Some(rbf) => rbf.fees.amount().to_sat() as i64 * -1000,
```

Comment:
Do we have a millisats to sats constant? Might be more clear here.

Also, I assume the negative here is what makes the `PendingTransaction` a liability?

Reply:
I found this whole function a little confusing as well with the i64 and negatives ...

Reply:
Negatives are liabilities, positives are assets, everything is millisat denominated so we need to multiply by 1000.

Could definitely use comments.

Reply:
Oh. Makes me think we should `TryFrom` instead of `as` to eliminate any chance for silent integer conversion issues.

Reply:
Don't think it should be an issue for `u64` as we'd need to have more sats than could ever exist :smile:

Reply:
With msats and some feerate calculations, maybe, maybe it would be possible to theoretically? Anyway - I think it's just for the habit of it. :D